src/EditorFeatures/CSharpTest/CodeActions/ConvertLinq/ConvertForEachToLinqQueryTests.cs

```diff
@@ -3864,7 +3903,7 @@ class C
 {
     void M(IEnumerable<int> nums)
     {
-        int c = (nums.AsQueryable().Select(n1 => n1)).Count();
+        int c = (nums.AsQueryable()).Count();
```

Comment:
nit: can you file bug about removing these unnecessary parens.

Reply:
Filed as https://github.com/dotnet/roslyn/issues/39727